tests/tracking/test_tracking.py

```diff
@@ -142,13 +142,12 @@ def test_set_experiment_with_deleted_experiment():
 
 def test_list_experiments():
     def _assert_exps(ids_to_lifecycle_stage, view_type_arg):
-        result = set(
-            [
-                (exp.experiment_id, exp.lifecycle_stage)
-                for exp in client.list_experiments(view_type=view_type_arg)
-            ]
-        )
-        assert result == set([(exp_id, stage) for exp_id, stage in ids_to_lifecycle_stage.items()])
+        result = [
+            (exp.experiment_id, exp.lifecycle_stage)
+            for exp in client.list_experiments(view_type=view_type_arg)
+        ]
+
+        assert result == [(exp_id, stage) for exp_id, stage in ids_to_lifecycle_stage.items()]
```

Comment:
```suggestion
        assert result == list(ids_to_lifecycle_stage.items())
```

nit: can we simplify this line?